zeroize/Cargo.toml

```diff
@@ -17,6 +17,7 @@ keywords = ["memory", "memset", "secure", "volatile", "zero"]
 edition = "2018"
 
 [dependencies]
+serde = { version = "1.0", optional = true }
```

Comment:
```suggestion
serde = { version = "1.0", default-features = false, optional = true }
```

Reply:
We generally try to enable the minimum number of features needed for a particular purpose.

Using `default-features = false` makes sense for both binaries and libraries, and is a requirement to use a prospective `serde` feature in `no_std` environments. See: https://serde.rs/no-std.html

Reply:
Done.